package.json

```
@@ -48,6 +48,6 @@
     "jest": "^29.7.0"
   },
   "engines": {
-    "node": "^20"
+    "node": "^20 || ^22"
```

I'd recommend only LTS versions if you can

```suggestion
    "node": "^20.9.0 || ^22.11.0"
```

---

Can you replace `20` with `ltd/jod` inside [**.nvmrc**](https://github.com/nhsuk/nhsuk-prototype-kit/blob/main/.nvmrc) too?

---

@colinrotherham done